src/libraries/Microsoft.Extensions.Configuration.Xml/tests/XmlConfigurationTest.cs

```diff
@@ -432,7 +432,8 @@ public void XmlConfiguration_Does_Not_Throw_On_Optional_Configuration()
             var config = new ConfigurationBuilder().AddXmlFile("NotExistingConfig.xml", optional: true).Build();
         }
 
-        [ConditionalFact]
+        [Fact]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/37669", TestPlatforms.Browser)]
         [FrameworkSkipCondition(RuntimeFrameworks.Mono)]
```

Comment:
I think this can be removed as well, it doesn't do anything useful anymore since this is about the old .NET Framework-based Mono. We can do that in a separate PR, filed https://github.com/dotnet/runtime/issues/38878.